### ISSUE TITLE
DBProfileViewController stopped working all of a sudden

### ISSUE THREAD

**Issue by Jackson0111** [DevonBoyer/DBProfileViewController#21]
Hi! I did a pod update, and I saw bunch of compile errors that just appeared out of nowhere. I'm not sure if this has anything to do with the library or my pod is not updated or something else. But take a look and please let me know if you know what's going on.

![screen shot 2016-05-20 at 2 44 26 am](https://cloud.githubusercontent.com/assets/13260808/15419587/0e21a752-1e35-11e6-81fb-7fde20abc885.png)

**Comment by devonboyer** [comment]
Thanks for letting me know!

Did you update CocoaPods to 1.0.0? I was having many similar problems myself after updating CocoaPods (http://stackoverflow.com/questions/37222742/updating-project-to-cocoapods-1-0-0-resulting-in-many-errors)

The solution that finally worked was to remove `use_frameworks!` from the Podfile since it appears to be bugged in CocoaPods 1.0.0 (which I know is not ideal since this prevents using Swift pods).

I am still searching for a better solution to this problem. I will put some time in today to see if I can fix things but for right now I'm blaming CocoaPods 1.0.0.

**Comment by Jackson0111** [comment]
Yes I did update to 1.0.0, after hours of trying to fix this issue, I gave up and started a new project and copy paste all my old code. Somehow that seems to work now, haha. Thanks!

**Comment by rsacchi** [comment]
Having the same issue, but using CP 0.39.0
Cannot remove use_frameworks! since I am using swift libraries.
